empress/clumpr/recon_vis/tree.py

```diff
@@ -19,14 +19,10 @@ def __init__(self, name):
         self.parent_node = None     # Node:  parent Node or None
         self.layout = None          # NodeLayout object: layout of this node
 
-    # The @property decorator allows this to be called as .is_leaf rather than .is_leaf()
-    @property
```

Comment:
I don't think this should be a property because it's not a way of accessing an instance variable. For example, `string.isdigit()`, `string.islower()`, `string.upper()` in are not properties.

Reply:
If there is a variable named `_is_leaf`, then it would make sense to make `is_leaf` a property.